embedded-bins/Makefile

```diff
@@ -5,6 +5,7 @@ kubernetes_version = 1.20.4
 kine_version = 0.6.0
 etcd_version = 3.4.14
 konnectivity_version = 0.0.14
+crictl_version = 1.20.0
```

Comment:
that gives us about +40 megabytes to the binary size:\ is it somehow possible to go without that tool? for example we can do the same unix socket requests or even use the tool but by importing code as a library

Reply:
I agree, but I think this is good enough for this release.

Reply:
I agree with both: For future releases we should aim to remove the dependency in an additional binary.